classes/PodsAdmin.php

```diff
@@ -1360,9 +1366,38 @@ public function admin_setup_edit_proto( $duplicate, $obj ) {
 
 		wp_localize_script( 'pods-dfv', 'podsAdminConfig', $config );
 
+		$this->admin_js_dfv_config();
+
 		pods_view( PODS_DIR . 'ui/admin/setup-edit-proto.php', compact( array_keys( get_defined_vars() ) ) );
 	}
 
+	/**
+	 * Handle setting up the JS DFV config.
+	 *
+	 * @since 2.8.0
+	 */
+	protected function admin_js_dfv_config() {
+		$config = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 1fa11c14814306f4b175528869ded911</sub>